src/components/client/actions/get-search-data.tsx

```diff
@@ -0,0 +1,30 @@
+"use server";
+
+import { indexAndSearch, searchIndexForData } from "@/helpers/search-data";
+import { SearchDataParams, SearchIndexData } from "@/helpers/types";
+import searchDataIndex from "../../../../public/search-index.json";
+
+type SearchDataType = {
+  entries: SearchIndexData[];
+};
+
+async function getSearchDataFromDirectory({ path, query }: SearchDataParams) {
+  const directory = `public/static/static${path ? "/" + path : "/"}`;
```

Comment:
do we really need `static/static`?

Reply:
This is the path to the files. This is just to generate the json one time.

Reply:
I can remove the function since we ain't using it ATM. This just searches the directories instead of using the search index. It was the 1st implementation I did